microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/integration/FailSafeCDIHealthDynamicBaseTest.java

```diff
@@ -21,7 +21,7 @@
 /**
  * Testclass to test {@code /health}, {@code /health/live}, {@code /health/ready} endpoints. Liveness and readiness probes
  * are configured via MP Config properties at the beginning and changed later in test execution. Tests in the testclass
- * expect no reload operation after changing the values.
+ * expect no reload operation after changing the values (during the test execution).
```

Comment:
Here is a better version - IMHO - than the version I commented on previously for `FailSafeCDIHealthBaseTest` class Javadoc (last sentence).